app/controllers/api/v2/registration_commands_controller.rb

```diff
@@ -15,7 +15,7 @@ class RegistrationCommandsController < V2::BaseController
         param :smart_proxy_id, :number, desc: N_("ID of the Smart Proxy. This Proxy must have enabled both the 'Templates' and 'Registration' features")
         param :setup_insights, :bool, desc: N_("Set 'host_registration_insights' parameter for the host. If it is set to true, insights client will be installed and registered on Red Hat family operating systems")
         param :setup_remote_execution, :bool, desc: N_("Set 'host_registration_remote_execution' parameter for the host. If it is set to true, SSH keys will be installed on the host")
-        param :jwt_expiration, :number, desc: N_("Expiration of the authorization token (in hours)")
+        param :jwt_expiration, :number, desc: N_("Expiration of the authorization token (in hours), -1 means 'unlimited'.")
```

Comment:
Should we explicitly mention all the possibilities? -2..,-1,0, 1, max value(999999)

Reply:
you mean like a validation? 
To align it with UI, its a good idea to set the max value to 999999.